std_msgs/Header header

float32 speed
float32 lane_width
float32 lane_offset
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity_local_3d
geometry_msgs/Vector3 angular_acceleration_local_3d
int32 lane_number
int32 navigation_segment